index.js

```diff
@@ -4,27 +4,23 @@ module.exports = function (img, selection_canvas, onCrop) {
   if('function' === typeof selection_canvas)
     onCrop = selection_canvas, selection_canvas = null
 
-  var c2 = selection_canvas = selection_canvas || h('canvas.hypercrop__selection', {width: 512, height: 512})
-
   onCrop = onCrop || function () {}
 
- var width = img.width, height = img.height
+  var c2 = selection_canvas = selection_canvas || h('canvas.hypercrop__selection', {width: 512, height: 512})
```

Comment:
`selection_canvas` isn't used past this point. Maybe drop `c2` and use `selection_canvas` throughout?